components/productCard/ProductImage.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="product-image">
+    <a :href="url" :title="title">
+      <img width="200" height="200" :src="cloudinaryImg" :alt="title" />
+    </a>
+  </div>
+</template>
+
+<script lang="ts">
+import Component from 'vue-class-component'
+import { Prop, Vue } from 'vue-property-decorator'
+
+@Component({})
```

Comment:
What is this?

Reply:
I saw you used it in here: https://github.com/coopjakob/kenora-thistles/blob/master/src/components/HelloWorld.vue so I tried using it here as well

Reply:
Oh, I remember now, but it was just a template. Do you think we should create components using that kind of structure?